.circleci/real_config.yml

```diff
@@ -1694,19 +1694,58 @@ jobs:
             - webui/react/build
 
   test-e2e-react:
-    docker:
-      - image: mcr.microsoft.com/playwright:v1.40.0-focal
+    machine:
+      image: <<pipeline.parameters.machine-image>>
     resource_class: large
     steps:
+      - run: echo 'export PW_SERVER_ADDRESS="http://localhost:3001"' >> $BASH_ENV # DNJ TODO -remove when tested
+      - run: echo 'export PW_USER_NAME=admin' >> $BASH_ENV
+      - run: echo 'export PW_PASSWORD=' >> $BASH_ENV
```

Comment:
any risk here?

Reply:
no, this sets up a local cluster on the circle-ci runner. So if you ssh to the circle executor, environment is open, but that's what ssh keys are for. This is exposed to the public internet.